lametro/templates/partials/past_event_item.html

```diff
@@ -0,0 +1,46 @@
+<div class="row">
+  <br/>
+  <div class="col-xs-4">
+    {% if event.status == 'cancelled' %}
+      <strike>
+    {% endif %}
+      <small>
+        <i class="fa fa-fw fa-calendar-o"></i>
+        <span class="non-mobile-only">
+          {{event.start_time | date:"D"}}
```

Comment:
Ditto consistency – can you use spaces inside the curly braces?

Reply:
Sure. Since I copied most of this code from `event_item.html`, would you like me to add spaces in that file as well? There are both tags with and without spaces in there as well.

Reply:
So long as new code looks ok, I'm fine with leaving the old as is.